src/einsteinpy/symbolic/ricci.py

```diff
@@ -28,6 +29,8 @@ def __init__(self, arr, syms, config="ll", parent_metric=None):
         parent_metric : ~einsteinpy.symbolic.metric.MetricTensor or None
             Corresponding Metric for the Ricci Tensor.
             Defaults to None.
+        name : String
+            Name of the Tensor. Defaults to "RicciTensor". To be modified, only when the Tensor object is modified in some way
```

Comment:
You should add a TODO ? This is not the right way of doing this.

Reply:
> Also, I request you to write useful PR titles. So its not a nightmare for me while writing changelog.

Sure. Will keep this in mind.

Reply:
This was not a ToDo. I meant, the `name` will only be modified, when the tensor is modified in some way (Lorentz Transform or Config Change). I have removed this portion, regardless, as `name` is user-modifiable and we are not tracking changes to the tensor anymore.